charts/dispatch/charts/identity-manager/templates/deployment.yaml

```diff
@@ -28,12 +28,19 @@ spec:
           imagePullPolicy: {{ default .Values.global.pullPolicy .Values.image.pullPolicy }}
           args:
             - "--cookie-name=_oauth2_proxy"
-            - "--policy-file=/etc/config/policy.csv"
             - "--host=0.0.0.0"
             - "--port=8080"
+            - "--db-file={{ default .Release.Name .Values.global.db.release }}-{{ .Values.global.db.host }}.{{ default .Release.Namespace .Values.global.db.namespace }}:{{ .Values.global.db.port }}"
```

Comment:
What do we want to make a CLI arg vs. what goes in config.json?  I want us to be consistent across services.  I just took things out of CLI args for function manager.  I don't really care one way or the other, I just want us to be consistent.

Reply:
I still see command args being passed to the function manager's deployment https://github.com/vmware/dispatch/blob/master/charts/dispatch/charts/function-manager/templates/deployment.yaml#L31

Did you mean something else?